air/src/air.rs

```diff
@@ -119,7 +119,7 @@ pub trait AirBuilder: Sized {
         ExprExt: AbstractExtensionField<Self::Expr>,
         I: Into<ExprExt>,
     {
-        let xe: ExprExt = x.into();
```

Comment:
Whoops, this function should be taking `x: I` rather than `x: ExprExt`, then the `into()` makes sense

Reply:
Oh oops, clippy did that, I didn't mean to include it. But I've made that change!